### ISSUE TITLE
id3v22-tda.mp3 considered non-free

### ISSUE THREAD

**Issue by cypriani** [taglib/taglib#760]
Hi again,

The Debian FTP Masters have apparently [decided](https://bugs.debian.org/736203) that the file id3v22-tda.mp3 from the test suite looks suspicious. Please see [this issue on the pytaglib project](https://github.com/supermihi/pytaglib/issues/5).

This was addressed in [this pytaglib commit](https://github.com/supermihi/pytaglib/commit/082da74f4906c1c416090c09231c3cec36550239), and the replacement file can be fetched from there: https://github.com/supermihi/pytaglib/blob/master/tests/data/r2.mp3

I personnaly agree with the people involved that this is nitpicking, but since the pytaglib developper created a drop-in replacement, would you consider using it instead? This would save the job of repacking the tarball in the Debian package for each new taglib release. You can just replace the file and keep the name id3v22-tda.mp3, since the automated check is done on the file hash rather than the file name; hence no source code change will be required.

Thanks in advance,
Matteo

**Comment by TsudaKageyu** [comment]
I agree that the file looks suspicious and it's better to replace it. I'll do it soon.
